docs/source/reference/api/index.rst

```diff
@@ -5,6 +5,7 @@ API Reference
    :maxdepth: 2
 
    client/index
+   graviti_client
```

Comment:
should we build docs for it?
I think the code under `graviti` will not disturb the `tensorbay` both source code and docs.

But with the development of graviti, may be we need to keep two different docs alive.